packages/instrumentation-prisma-client/src/instrumentation.ts

```diff
@@ -45,20 +64,32 @@ export class PrismaClientInstrumentation extends InstrumentationBase {
     return function (original: () => any) {
       return function patchedRequest(this: any) {
         const args = arguments[0] as {
-          clientMethod: string;
+          action: string;
+          args: Map<String, String>;
```

Comment:
I'm not a Javascript/Typescript dev, so I don't really know if there's a better way of doing this...

I am learning a lot though.